_partials/jbds-is.adoc

```diff
@@ -2,11 +2,11 @@
 
 1.  Install the Java SE Development Kit (JDK), we recommend OpenJDK or
 Oracle JDK.
-2.  Download link:#{site.products['devstudio'].downloads['8.1.0.GA'].assets[site.products['devstudio'].default_download_artifact_type].artifacts[0].url}[JBoss Developer Studio 8.1.0.GA #{site.products['devstudio'].downloads['8.1.0.GA'].assets[site.products['devstudio'].default_download_artifact_type].artifacts[0].name}]
+2.  Download link:https://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-jar_universal.jar?tp=fuse[JBoss Developer Studio 8.1.0.GA]
```

Comment:
The link needs to be paramaterized so that the correct download manager is used, depending on what environment the site is being built for.

Do this by replacing:

     https://developers.redhat.com/download-manager/file

with:

     #{download_manager_file_base_url}